kubeflow/common/iap-ingress/Makefile

```diff
@@ -30,3 +30,5 @@ pod-reset:
 	# Kick the backend updater pod, because information might be outdated after the apply.
 	# https://github.com/kubeflow/gcp-blueprints/issues/160
 	kubectl --context=$(KFCTXT) -n istio-system delete statefulset backend-updater
+	# Kick the cloud-endpoints-enabler deployment
+	kubectl --context=$(KFCTXT) -n istio-system delete deployment cloud-endpoints-enabler
```

Comment:
We don't need to take action right now, but I am thinking if we can define a configMap variable `isScriptSuccessful` for the readiness of Ingress/IAP/Cloud-endpoint. Then these scripts can change configMap value after successfully update RequestAuthentication/BackendConfig/etc. As a result, we can write a script to read configmap `isScriptSuccessful`'s value, and run `make pod-reset` to turn down scripts if value is true.

Reply:
SGTM